### PR TITLE
Center map pin when zoomed in

### PR DESCRIPTION
**Pull request by alvihrhp**
1. Adding reload map function when dragging the map, it seems the mer…ged fail from branch #28, 
2. Fix centering the marker

### CODE REVIEW

**Comment by alvihrhp** [comment]
Hi @lawriecate, @reustle just noticed there is a bug when rendering the marker which is not centered, by default google maps render the marker on the top left so we need to center it by adding some css. Also, i've noticed that my PR from reload map hadn't got merged correctly and some functions are gone, i fix that too. Furthermore, i fix some design to make it more presentable. Thank you.

**Comment by lawriecate** [comment]
Thank you so much @alvihrhp ! Sorry I made the mistake during the merge.  Checking and will merge shortly!

**Review by lawriecate — APPROVED** [review]
LGTM, but please remove `yarn-error.log` or let's add to .git-ignore file

**Comment by alvihrhp** [comment]
@lawriecate don't merge it yet, because currently i'm working on a sub branch from this PR for mobile layout.

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "-": "^0.0.1",
+    "@turf/turf": "^6.5.0",
```
Comment (lawriecate) [inline package.json:17]: is this package used?

**Comment by lawriecate** [comment]
> @lawriecate don't merge it yet, because currently i'm working on a sub branch from this PR for mobile layout.

OK thank you!!

**Comment by alvihrhp** [comment]
Hi @lawriecate, I've fixed the "also modal/search components overlap" making it more similar to Figma design. But, i don't understand by this comment "Mobile layout is too high-resolution (possibly from metrics section)", because when i inspect it, the styling is the same as Figma design

**Comment by lawriecate** [comment]
> Hi @lawriecate, I've fixed the "also modal/search components overlap" making it more similar to Figma design. But, i don't understand by this comment "Mobile layout is too high-resolution (possibly from metrics section)", because when i inspect it, the styling is the same as Figma design

Ah sorry @alvirhrhp I should have described better, I think from when I added the metrics component the width of the screen became too wide as shown in this screenshot .  Maybe you've fixed already but let me know if it makes sense!:-

IMG_55677311A240-1

**Comment by alvihrhp** [comment]
> > Hi @lawriecate, I've fixed the "also modal/search components overlap" making it more similar to Figma design. But, i don't understand by this comment "Mobile layout is too high-resolution (possibly from metrics section)", because when i inspect it, the styling is the same as Figma design
> 
> Ah sorry @alvirhrhp I should have described better, I think from when I added the metrics component the width of the screen became too wide as shown in this screenshot . Maybe you've fixed already but let me know if it makes sense!:-
> 
> IMG_55677311A240-1

Hi @lawriecate, i just added media query and make the mobile design similar to the Figma design. But, maybe because i added media query, it automatically fix the width of the metrics. From my point of view it's make sense and if you wanna merge it, you can go ahead. Than you !

**Review by lawriecate — APPROVED** [review]

**Comment by lawriecate** [comment]
> > > Hi @lawriecate, I've fixed the "also modal/search components overlap" making it more similar to Figma design. But, i don't understand by this comment "Mobile layout is too high-resolution (possibly from metrics section)", because when i inspect it, the styling is the same as Figma design
> > 
> > 
> > Ah sorry @alvirhrhp I should have described better, I think from when I added the metrics component the width of the screen became too wide as shown in this screenshot . Maybe you've fixed already but let me know if it makes sense!:-
> > IMG_55677311A240-1
> 
> Hi @lawriecate, i onlay added media query and make the mobile design similar to the Figma design. But, maybe because i added media query, it automatically fix the width of the metrics. From my point of view it's make sense and if you wanna merge it, you can go ahead. Than you !

Great thank you so much!! I'll merge it :) 🚀

**Pull request merged**